src/data.js

```diff
@@ -0,0 +1,35 @@
+const Pieces = [
```

Comment:
This content is part of another story. You could add the FILES and RANKS arrays to this?